### PR TITLE
Implement missing labels for post type registration (fixes #808)

### LINKED ISSUE #808: Implement new labels introduced in WordPress 5.0 for our custom post types
## Issue Overview

With WordPress 5.0 comes the introduction of [new i18n labels for custom post types](https://make.wordpress.org/core/2018/12/05/new-post-type-labels-in-5-0/).  It should be trivial to get those added in our custom post type registration (will also need to do this for the people add-on).

We will have to consider whether this is something that is back-compat or not but on the surface I think it should be fine to add them, earlier versions of WP just won't implement the new labels.

Time to implement would be very minimal.

### PR DESCRIPTION
**Pull request by nerrad**
## Problem this Pull Request solves

This pull implements a number of missing labels for post type registration for our custom post types.  Also adds missing `/* Translators: */` comments.

This has minimal impact on existing ui because for the most part we've already customized forward facing labels in our admin pages but where this will be helpful is when post types are used outside of our native routes.  It is possible some themes will use these labels for things like archive views etc where this will be picked up as well.

## How has this been tested

* [x] As stated, this has close to nil impact on existing behaviour, I just visited various admin pages for our cpts (Events, Venues, Attendees) and didn't notice anything out of the ordinary.

## Checklist

* [ ] I have read the documentation relating to systems affected by this pull request, see https://github.com/eventespresso/event-espresso-core/tree/master/docs
* [ ] User input is adequately validated and sanitized
* [ ] all publicly displayed strings are internationalized (usually using `esc_html__()`, see https://codex.wordpress.org/I18n_for_WordPress_Developers)
* [ ] My code is tested.
* [ ] My code follows the Event Espresso code style.
* [ ] My code has proper inline documentation.
* [ ] My code accounts for when the site is in Maintenance Mode (MM2 especially disallows usage of models)

### CODE REVIEW

File: core/domain/services/custom_post_types/RegisterCustomPostTypes.php [hunk 1]
```diff
@@ -168,38 +168,112 @@ private function getLabels($singular_name, $plural_name, $singular_slug, $plural
...
+                $singular_name
+            ),
+            'attributes' => sprintf(
+                /* Translators: Post Type Label */
+                esc_html_x('%s Attributes', 'Event Attributes', 'event_espresso'),
+                $singular_name
+            ),
+            'insert_into_item' => sprintf(
+                /* Translators: Post Type Label */
+                esc_html_x('Insert into this %s', 'INsert into this Event', 'event_espresso'),
```
Comment (Pebblo) [inline core/domain/services/custom_post_types/RegisterCustomPostTypes.php]: Capitalization - INsert.

File: core/domain/services/custom_post_types/RegisterCustomPostTypes.php [hunk 2]
```diff
@@ -168,38 +168,112 @@ private function getLabels($singular_name, $plural_name, $singular_slug, $plural
...
+                /* Translators: Post Type Label */
                 esc_html_x('All %s', 'All Events', 'event_espresso'),
                 $plural_name
             ),
             'view_item'          => sprintf(
+                /* Translators: Post Type Label */
                 esc_html_x('View %s', 'View Event', 'event_espresso'),
                 $singular_name
             ),
+            'view_items' => sprintf(
```
Comment (Pebblo) [inline core/domain/services/custom_post_types/RegisterCustomPostTypes.php]: Spacing is different here than the above items?
(phpcs didn't highlight this so just checking)

File: core/domain/services/custom_post_types/RegisterCustomPostTypes.php (hunk 1 above)
Comment (nerrad) [reply core/domain/services/custom_post_types/RegisterCustomPostTypes.php]: woops good catch.

File: core/domain/services/custom_post_types/RegisterCustomPostTypes.php (hunk 2 above)
Comment (nerrad) [reply core/domain/services/custom_post_types/RegisterCustomPostTypes.php]: That's just an personal preference, not necessary.

**Comment by joshfeck** [comment]
I gave this branch a quick test (visited various CPT routes) and it's ready for merging.

**Pull request merged**